mtp_api/apps/prison/migrations/0021_prison_use_nomis_for_balances.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.0.13 on 2020-10-23 11:15
+
+from django.db import migrations, models
+
+
+def flag_private_estate_as_not_using_nomis_for_balances(apps, schema_editor):
```

Comment:
Are we setting Birmingham manually? If so which ticket is that a part of?

Reply:
No ticket, manual django admin task will probably be triggered by receiving a CSV from Birmingham if nothing else (changing the use_nomis_for_balance setting on a prison)

Reply:
birmingham is still unconfirmed. there's probably no specific ticket for it. there will likely be other manual setup – like the establishments which cannot provide these reports